
# Intersection
left_check_debug: False
left_check_length: 45.0
left_check_x_transform: 12.0
